docs/validate-stable-rc-builds.md

```diff
@@ -0,0 +1,237 @@
+# Stable rc review validation process
+Following steps to be performed when a stable rc review pushed on stable mailing
+list
+
+## Wait for stable rc review email
+Greg KH pushes on to stable rc trees and creates a tag
+then sends out email on stable ML with the list of patches included in that
+branch.
+
+stable-rc tree link:
+
+*https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable-rc.git*
+
+stable-rc review email example link,
+
+*https://www.spinics.net/lists/stable/msg320559.html*
+
+Longterm release kernels details
+
+|Version|Maintainer|Released|Projected EOL|
+|-------|:---------|:-------|:-----------:|
+|4.19|Greg Kroah-Hartman|2018-10-22|Dec, 2020|
+|4.14|Greg Kroah-Hartman|2017-11-12|Jan, 2024|
+|4.9|Greg Kroah-Hartman|2016-12-11|Jan, 2023|
+|4.4|Greg Kroah-Hartman|2016-01-10|Feb, 2022|
+
+one must have to subscribe to the stable mailing list.
+You may check your inbox with given filters to identify the stable rc review
+for a given branch.
+
+## Build Test
+### Check for build is PASS
+open lkft.linaro.org website and check the build status
+From *https://lkft.linaro.org* you will re-directed to each branch builds for
+each device. If you see build status is in progress wait for the
```

Comment:
Or if there are builds queued.

Also, 4.4 has two jobs: one for HiKey and one for the rest.

Reply:
All 4.4 is also listed on lkft.linaro.org
so no need to specify about 4.4 hikey build